presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ScalarFromAnnotationsParser.java

```diff
@@ -99,11 +106,31 @@ private static List<ScalarHeaderAndMethods> findScalarsInFunctionSetClass(Class<
         return builder.build();
     }
 
+    private static Optional<ScalarStatsHeader> getScalarStatsHeader(Method annotated)
+    {
+        Optional<ScalarStatsHeader> scalarStatsHeader;
+        ScalarFunctionConstantStats statsCalculator = annotated.getAnnotation(ScalarFunctionConstantStats.class);
+        List<java.lang.reflect.Parameter> params = Arrays.stream(annotated.getParameters()).filter(x -> x.getAnnotation(SqlType.class) != null).collect(Collectors.toList());
+        // Map of (function argument position index) -> (ScalarPropagateSourceStats annotation)
+        Map<Integer, ScalarPropagateSourceStats> argumentStatsResolver = new HashMap<>();
+
+        IntStream.range(0, params.size())
+                .filter(x -> params.get(x).getAnnotation(ScalarPropagateSourceStats.class) != null)
```

Comment:
same comment on variable name. There are a few other cases in this class. Please fix them as well